news/_posts/2021-04-06-we-decided-to-use-typescript.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "We decided to use TypeScript!"
+subtitle: "Why did we do this?"
+date: 2021-04-06 17:00:00
+authors: [NeOMakinG]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [news, core]
+---
+
+Recently, the PrestaShop core team decided to use TypeScript (TS) on the project. [We recently submited an Architecture Decision Records (ADR)](https://github.com/PrestaShop/ADR/pull/19) and a [Proof of Concept (POC)](https://github.com/PrestaShop/PrestaShop/pull/23221) in order to use TypeScript on every development we do.
```

Comment:
It's not merged :p so it's in discussion

Reply:
an ADR and a POC is all about discussion isn't it ? :p